docs/cloud/ui/flow.md

```diff
@@ -0,0 +1,25 @@
+# Flow
+
+## Overview
+
+![](/cloud/ui/flow-overview.png)
+
+The flow overview summarizes each flow's recent behavior. It is a dashboard for a single workflow.
+
+## Schematic
+
+![](/cloud/ui/flow-schematic.png)
+
+The flow schematic shows an interactive overview of all the tasks in the flow and their dependency structure.
+
+## Versions
+
+![](/cloud/ui/flow-versions.png)
+
+The "versions" page shows a history of all versions of this flow. 
+
+## Run
+
+![](/cloud/ui/flow-run.png)
+
+Kick off a new run of your flow, providing parameter values if necessary.
```

Comment:
Could you mention here that you can also provide a Flow Run name for easy searching / documentation?